packages/rancher-monitoring/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: beaa76a4ac26584c6cc77c27bc0d40769fb98b26
-version: 103.0.4
+version: 103.0.5
```

Comment:
This change strikes me as worthy of a minor (rather than patch) version bump. @lucasmlp @nicholasSUSE what do you think?

Reply:
I've updated the version to `103.1.0+up45.31.1`